### PR TITLE
docs: release notes 2.4

### PR DESCRIPTION
**Pull request by graikhel-intel**
## PR Checklist

- [ ] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [ ] Have viewed and verified edits in rendered form ensuring proper formatting

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -11,135 +11,117 @@
...
 
-With this release, UI-Toolkit-Angular supports Angular v13. This breaking change is indicated by the major version transition to 3.x.x. If you are still on Angular 12, stick with UI-Toolkit-Angular 2.0.5 until you migrate Angular v13.
+:material-star:** Coming Soon: Set AMT Features with AMT Profile**
 
-:material-auto-fix:** Improvement: UI Toolkit - KVM mouse alignment when scrolled**
+In our next release, we will be supporting the ability to set AMT Features as part of the AMT Profile during activation. This includes settings such as IDE Redirection, and User Consent Mode for KVM.  This feature will eliminate the need for setting these features post-activation.  While we wanted to complete this feature for this release, we weren't quite able to get everything in. However, the database changes to support it are included with this release. If you are migrating from v2.3.0 version of the toolkit, you'll need to add the following columns to the `profiles` table in the RPS Database.
```
Comment (Kimberly-Davis) [inline docs/release-notes.md:48]: In our next release, we will be support the ability to set AMT Features as part of the AMT Profile during activation. This includes settings such as IDE Redirection and User Consent Mode for KVM.  This feature eliminates the need for setting these features post-activation. However, the database changes to support this feature are included with this release. If you are migrating from v2.3.0 version of the toolkit, you'll need to add the following columns to the `profiles` table in the RPS Database:

**Review by Kimberly-Davis — CHANGES_REQUESTED** [review]

File: docs/release-notes.md [hunk 2]
```diff
@@ -11,135 +11,117 @@
...
+*The Open AMT Cloud Toolkit Team*
 <br>
 <p class="divider"></p>
-
 ## What's New?
 
-:material-star:** Customer Request: RPC-Go as a library**
+:material-star:** Customer Request: Support for configuring Wi-Fi only platforms**
+
+With this release, Customers can now provision and manage vPro AMT platforms which only have a vPro AMT supported Wi-Fi adapter. These platforms can only be activated in CCM. For ACM activation, manual touch through MEBx is required.
```
[the remark this answers is not in the document]
Comment (graikhel-intel) [reply docs/release-notes.md]: done

File: docs/release-notes.md (hunk 1 above)
Comment (graikhel-intel) [reply docs/release-notes.md]: done

File: docs/release-notes.md [hunk 3]
```diff
@@ -11,135 +11,117 @@
...
+:material-star:** Customer Request: Short lived JWT for redirection sessions**
 
-:material-auto-fix:** Improvement: RPC-Go Refactoring**
+MPS can now issue a short-lived JWT (default 5 min) that can be used to authenticate Redirection sessions between the console and MPS so that Redirection sessions can only be initiated by an authenticated and authorized user. This short-lived token can be configured using "redirection_expiration_time" property in `.mpsrc` file within MPS or overridden with ENV. You can find more details about the API under User Authentication section of MPS APIs.
 
-Our Go version of RPC underwent significant refactoring to eliminate all C/C++ code. By transitioning to 100% Go code, we removed the requirement of the GCC toolchain to build RPC as an application. Developers can cross-compile RPC on Linux or Windows and use it on any Go-supported OS (e.g., We tested on Fedora, and it worked without any code changes). The build time is greatly reduced. This version of RPC is still in beta.
+:material-new-box:** Feature: HTTP Entity Conflict Support in RPS**
 
-:material-star:** Customer Request: GET Device by hostname**
```
[the remark this answers is not in the document]
Comment (graikhel-intel) [reply docs/release-notes.md:33]: done

**Review by Kimberly-Davis — APPROVED** [review]

**Pull request merged**